### DISCUSSION TITLE
Switching from vxlan to wireguard-native

### DISCUSSION THREAD

**Discussion by hvojdani** [k3s-io/k3s#8868] [Q&A] [answered]
I have a multi master with 2 node k3s instance.  have not configured flannel-backend and both node shows vxlan is used.
both node have public visible IP.  I want to switch to wireguard-native and installed packages required by it. 
I have a question **just updating service config and restarting them is enough?**
should I specify iface or any other parameters?
1-3 minutes downtime is acceptable. if it is more complicated  please describe how to do it. I have hardened both servers by firewall rules.

**Comment by brandond** [comment, marked as answer]
Assuming everything else is in place, then all you should need to do is update the flannel-backend flag on the servers, and then restart all the nodes.

If you have a non-production environment, I would recommend doing it there first to make sure you have all the correct binaries and firewall rules in place.
